elastic/conf.yaml.example

```diff
@@ -15,6 +15,9 @@ instances:
   # This parameter was also called `is_external` and you can still use it but it
   # will be removed in version 6.
   #
+  # If you enable the "index_stats" flag, you will collect metrics for index 
+  # level metrics.
```

Comment:
Nit: can we change this to something like "If you enable the "index_stats" flag, you will collect metrics about individual indexes.